packages/lib/src/vis/utils.ts

```diff
@@ -71,10 +71,9 @@ export function getSizeToFit(
 
   const availableRatio = width / height;
 
-  return {
-    width: availableRatio > ratioToRespect ? height * ratioToRespect : width,
-    height: availableRatio < ratioToRespect ? width / ratioToRespect : height,
-  };
+  return availableRatio > ratioToRespect
+    ? { width: height * ratioToRespect, height }
+    : { width, height: width / ratioToRespect };
```

Comment:
I found this part not very readable when coming back to it. I hope it's better now.